include/CppUTest/CommandLineArguments.h

```diff
@@ -72,13 +73,15 @@ class CommandLineArguments
     bool listTestGroupAndCaseNames_;
     bool runIgnored_;
     int repeat_;
+    unsigned int shuffle_; // 0: shuffling disabled, 1: random seed, other values: specific random seed
```

Comment:
Can you use enum values of #defines for these rather than adding a comment?

Reply:
Fixed using defines